mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -108,5 +108,28 @@
     <string name="mozac_feature_addons_not_yet_supported_caption">Firefox add-on technology is modernizing. These add-ons use frameworks that are not compatible with Firefox 75 &amp; beyond.</string>
     <!-- This is the caption for the add-on installation progress overlay -->
     <string name="mozac_add_on_install_progress_caption">Downloading and verifying add-on…</string>
-
+    <!-- Error shown when something unexpected happened while trying to get the add-on list from the server -->
+    <string name="mozac_feature_addons_failed_to_query_add_ons">Failed to query Add-ons!</string>
+    <!-- Text shown after successfully installed an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_successfully_installed">Successfully installed: %1$s</string>
+    <!-- Text shown after failed to install an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_failed_to_install">Failed to install %1$s</string>
```

Comment:
@Amejia481: Just want to check it's on purpose that the previous (success) string finishes by a colon, while this one (failed) does not. Should we aim for consistency between both?
This comment actually also applies to all the other strings below.
Should we drop the colon from this string, or maybe add it everywhere?

Reply:
Thank you for caching this, the real intention was to drop the colon in all the strings. I will put a pr for updating it.

Reply:
https://github.com/mozilla-mobile/android-components/pull/5711